src/components/see-more-titles/see-more-titles.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { classnames } from "../../helpers";
+
+import * as styles from "./see-more-titles.module.scss";
+
+const SeeMoreTitles = ({ onClick, className, children}) => {
```

Comment:
I'm not sure, why do you define this component?

Reply:
I have defined that component only to show two links with an icon, should I remove it?